ssl/ssl_lib.c

```diff
@@ -5947,3 +5947,17 @@ int SSL_CTX_set0_tmp_dh_pkey(SSL_CTX *ctx, EVP_PKEY *dhpkey)
     ctx->cert->dh_tmp = dhpkey;
     return 1;
 }
+
+uint16_t SSL_get_grease_value(SSL *s,
+                              SSL_GREASE_INDEX index) {
+    RAND_bytes(s->grease_seed, sizeof(s->grease_seed));
```

Comment:
Should this be conditionally filled?  I.e. should calling `SSL_get_grease_value(s, 1)` twice return the same value?

Reply:
I initially had this conditionally filled with the boolean flag being stored in the SSL struct. Is there a good place in this code to set this as 0 or do a memset of the whole struct to 0? I kinda took out the conditional because I thought maybe the called could handle the boolean flag it it's not suitable to be in SSL struct.

Reply:
Added the boolean flag